Update game (#1)
rajendran-k/skills-review-pull-requests: README.md

</header>

<!--
  <<< Author notes: Step 5 >>>
  Start this step by acknowledging the previous step.
  Define terms and link to docs.github.com.
-->

> ## Step 5: Apply suggested changes

Author: ```suggestion
## Step 5: Apply suggested changes
```

> _Nicely done suggesting changes! :partying_face:_

Author: ```suggestion
_Nicely done suggesting changes!
```

</header>

<!--
  <<< Author notes: Step 4 >>>
  Start this step by acknowledging the previous step.
  Define terms and link to docs.github.com.
-->

## Step 4: Suggest changes

_Nice work reviewing that pull request :sparkles:_

Now that you have explored the different ways you can review a pull request it is time to learn how to use _suggest changes_.

**What is _suggest changes_?**: This feature enables you to recommend a change to a pull request that the author can commit with the push of a button.

### :keyboard: Activity: Suggest changes

> 1. On the pull request, click **Files changed**.

Author: ```suggestion
1. On the pull request, click **File change**.
```
